### PR TITLE
Newabbreves

### PR DESCRIPTION
**Pull request by abhiraj-ku**
This PR Adds Multiple Abbreviations

Closes #

### What new changes did you make? Tick all applicable boxes
- [ yes] Added new abbreviation
- [ ] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### Describe the new changes you added.4
New slangs/abbreves are as follows:

faq: Frequently Asked Questions
ctn: Can't Talk Now
cye : Check Your Email
iso: In Search Of
ot: On Time
otoh: On The Other Hand
pc: Personal Computer
rt: Real Time

### CODE REVIEW

**Comment by abhiraj-ku** [comment]
hey @Njong392 please Review The changes and suggest if any change is required

File: public/server/db/cfbr.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "definition": "Commenting For Better Reach"
```
Comment (Njong392) [inline public/server/db/cfbr.json:2]: Unfortunately, this already exists in the records. To make sure that an abbreviation doesn't already exist, you can look it up by searching on the site.

File: public/server/db/pc.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "definition":"Personal Computer"
```
Comment (Njong392) [inline public/server/db/pc.json:2]: Abbreve is an unofficial dictionary for slang. Unfortunately, abbreviations like these don't fit in. Hope you understand.

**Review by Njong392 — CHANGES_REQUESTED** [review]

File: public/server/db/pc.json (hunk 2 above)
Comment (abhiraj-ku) [reply public/server/db/pc.json:2]: Sure I make a PR again for this keeping in mind the changes suggested . Thanks

File: public/server/db/cfbr.json (hunk 1 above)
Comment (abhiraj-ku) [reply public/server/db/cfbr.json:2]: I didn't know how this came in this PR as i have created other branch for these abbreves

**Comment by abhiraj-ku** [comment]
Hey @Njong392  i Have deleted the files requested by you . I hope this Helps

**Comment by abhiraj-ku** [comment]
hey @Njong392 no updates yet ?

**Comment by Njong392** [comment]
Hello. Apologies for delaying the review. I was on holiday, but now I am back!

**Review by Njong392 — APPROVED** [review]
💣 Thank you so much for these additions!

**Comment by abhiraj-ku** [comment]
Hey @Njong392  i am happy to contribute on this . Looking forward to this project

**Pull request merged**